src/game_battlealgorithm.cpp

```diff
@@ -1104,16 +1103,17 @@ bool Game_BattleAlgorithm::Skill::Execute() {
 		}
 
 		if (this->healing) {
-			float mul = GetAttributeMultiplier(skill.attribute_effects);
-			if (mul < 0.5f) {
-				// Determined via testing, the heal is always at least 50%
-				mul = 0.5f;
-			}
-
-			effect +=
+			int effect = skill.power +
 				source->GetAtk() * skill.physical_rate / 20 +
 				source->GetSpi() * skill.magical_rate / 40;
-			effect *= mul;
+
+			// Determined via testing, the heal is always at least 50%
+			effect *= std::max(0.5f, GetAttributeMultiplier(skill.attribute_effects));
```

Comment:
here is the 50%-thing again

Reply:
Fixed

Reply:
iirc, I also saw this 50%-thing a few lines below again.